src/app/storage/sync.service.ts

```diff
@@ -68,18 +71,28 @@ const adapters: StorageAdapter[] = [new IndexedDBStorage(), GoogleDriveStorageAd
 let _getPromise: Promise<DimData> | undefined;
 let cached: DimData;
 
+let gapiLoaded = false;
+
 export const SyncService = {
   adapters,
   GoogleDriveStorage: GoogleDriveStorageAdapter,
 
   init() {
-    GoogleDriveStorageAdapter.init();
+    if (gapiLoaded) {
+      GoogleDriveStorageAdapter.init();
 
-    GoogleDriveStorageAdapter.signIn$.subscribe(() => {
-      // Force refresh data
-      console.log('GDrive sign in, refreshing data');
-      this.get(true).then(initSettings);
-    });
+      GoogleDriveStorageAdapter.signIn$.subscribe(() => {
+        // Force refresh data
+        console.log('GDrive sign in, refreshing data');
+        this.get(true).then(initSettings);
+      });
+    } else {
+      const apiScript = document.createElement('script');
+      apiScript.setAttribute('src', 'https://apis.google.com/js/api.js?callback=googleApiInit');
```

Comment:
I had some trouble finding the `googleApiInit` callback - I like that we're lazy loading the gapi now for users who haven't opted in yet!

Reply:
Ah, I see that `window.gapi_onload` should be called when this script loads to retrigger  SyncService.init() Maybe the callback isn't relevant.

Reply:
Yeah I should go back through this - or maybe even remove it from this PR. My intention is to make it so we don't load gdrive unless people want to use it, but I backed off from a more aggressive cutover so we're always trying it now. I should at least make it predicated on whether you have Google Drive enabled.